gui/robot_data.py

```diff
@@ -34,26 +33,28 @@ def get_float_value(packet):
     Returns: float of "value"
 
     '''
-    return float(get_values(packet))
+    return float(get_value(packet))
 
 
-def get_tuple_value(data):
+def get_values(data, num_inputs):
```

Comment:
make input list rather than tuple so we can store more than 2 values of data (need 3 for acceleration and coordinates)